cmd/cluster/connection-pooling/connection_pooling.go

```diff
@@ -104,6 +104,10 @@ func performConnectionPoolingOperation(operationName string, cmd *cobra.Command,
 
 func init() {
 	util.AddCommandIfFeatureFlag(ConnectionPoolingCmd, enableConnectionPoolingCmd, util.CONNECTION_POOLING)
+	enableConnectionPoolingCmd.Flags().String("cluster-name", "", "[REQUIRED] The name of the cluster to get details.")
```

Comment:
why is this required? It is added as part of a global flag. Ref: https://yugabyte.atlassian.net/browse/CLOUDGA-23029

Reply:
The description is misleading/incorrect too